index.html

```diff
@@ -60,66 +72,1830 @@
+                    <span class="red fs-6 text text-light"> COVER </span>
+                    <span> <a href="#" class="bg-light text-danger fs-5 text"> NEWS</a></span>
+                    <h4><a href="#">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></h4>
+                </div>
+            </div>
+            <div class="col-lg-3 col-md-3  magazine">
+                <div class="position-relative editor">
+                    <a href="#"><img src="img/pro-trump-mob-seizes-capitol.jpeg" class="img-fluid" alt="..."></a>
+                </div>
+                <div>
+                    <span class="red fs-6 text text-light"> COVER </span>
+                    <span> <a href="#" class="bg-light text-danger fs-5 text"> NEWS</a></span>
+                    <h4><a href="#">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></h4>
+                </div>
+            </div>
+            <div class="col-lg-3 col-md-3  magazine">
+                <div class="position-relative editor">
+                    <a href="#"><img src="img/pro-trump-mob-seizes-capitol.jpeg" class="img-fluid" alt="..."></a>
+                </div>
+                <div>
+                    <span class="red fs-6 text text-light"> COVER </span>
+                    <span> <a href="#" class="bg-light text-danger fs-5 text"> NEWS</a></span>
+                    <h4><a href="#">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></h4>
+                </div>
+            </div>
+            <div class="col-lg-3 col-md-3 magazine">
+                <div class="position-relative editor">
+                    <a href="#"><img src="img/pro-trump-mob-seizes-capitol.jpeg" class="img-fluid" alt="..."></a>
+                </div>
+                <div>
+                    <span class="red fs-6 text text-light"> COVER </span>
+                    <span> <a href="#" class="bg-light text-danger fs-5 text"> NEWS</a></span>
+                    <h4><a href="#">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></h4>
+                </div>
+            </div>
+            <div class="col-lg-3 col-md-3 magazine">
+                <div class="position-relative editor">
+                    <a href="#"><img src="img/pro-trump-mob-seizes-capitol.jpeg" class="img-fluid" alt="..."></a>
+                </div>
+                <div>
+                    <span class="red fs-6 text text-light"> COVER </span>
+                    <span> <a href="#" class="bg-light text-danger fs-5 text"> NEWS</a></span>
+                    <h4><a href="#">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></h4>
+                </div>
+            </div>
+        </div>
+        <!-- editors pick -->
+        <div class="n-title flex-xs jc-c ai-c d-flex justify-content-center mt-3  mb-3 margin-bottom fs-3 text"><span
+                class="icon-n "></span>EDITOR'S PICK</div>
```

Comment:
- [ ] images in your "editors pick" section is not correctly styled on medium screen size. Notice how the images are bigger. I took the screenshots on 856 px.
### original 👇 
![image](https://user-images.githubusercontent.com/64360974/114851741-d49e8a00-9dea-11eb-9dbb-052f7e73006b.png)

### yours 👇 
![image](https://user-images.githubusercontent.com/64360974/114851828-e8e28700-9dea-11eb-9d80-72a22cdb8bfa.png)